### PR TITLE
Allowed OTP-Release 18 in rebar.conf and made it compilable with it

### PR DESCRIPTION
**Pull request by NobbZ**
Tests were passing locally at least, now lets wait for travis!

### CODE REVIEW

**Comment by exterm** [comment]
It's a shame that coverage reporting apparently doesn't work with erlang 18. Maybe it will work again once we https://github.com/disco-framework/disco/issues/11 .

File: rebar.config [hunk 1]
```diff
@@ -60,7 +60,7 @@
   {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", "v1.4.2"}},
 
   %% for testing
-  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "v1.1"}},
+  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {branch, "master"}}}, %"v1.1"}},
```
Comment (exterm) [inline rebar.config]: branch master sounds dangerous. Maybe you could specify a commit?

I think there was some way to generate and use a rebar lock file, that would also be a possible solution.

**Comment by exterm** [comment]
Thank you for the PR. :+1:

**Comment by exterm** [comment]
Hmm, looks as if travis integration is broken. I'll take a look at it.

**Comment by NobbZ** [comment]
I will experiment with a specific commit this evening and update the PR
accordingly. Perhaps I might find to do some deeper research until then
because of the rebar + coverage related thing. Otp 18 itself does have the
necessary application available.

Philip Müller <notifications@github.com> schrieb am Mi., 27. Jan. 2016
14:38:

**Comment by exterm** [comment]
great :+1:

**Comment by exterm** [comment]
Strange, but closing and reopening the PR was the easiest way to retrigger the travis build.

**Comment by NobbZ** [comment]
> Maybe it will work again once we upgrade rebar to rebar3 ( #11 ).

It seems as if it doesn't. I'm doing some experiments with rebar3 on a separate branch, but I do get the same error, so I filed an issue over at rebar/rebar#1040. They told me it is a bug in OTP 18 and linked to an upstream PR, which might be in OTP 18.3 (erlang/otp#885)

**Comment by exterm** [comment]
Thank you for the great research! I made an [issue](https://github.com/disco-framework/disco/issues/14) out of that. 

Thank you again for the PR, too!

**Pull request merged**